src/doc/CreateHighlightDialog.js

```diff
@@ -79,8 +79,8 @@ class CreateHighlightDialog extends CreateAnnotationDialog {
     constructor(parentEl, config = {}) {
         super(parentEl, config);
 
-        this.allowHighlight = !!config.allowHighlight || true;
-        this.allowComment = !!config.allowComment || true;
+        this.allowHighlight = config.allowHighlight || false;
```

Comment:
Also @JustinHoldstock thoughts on disabling these buttons by default? It feels weird to enable buttons if no config is passed in, however I guess that could result in an empty dialog showing up if nothing is passed in...

Reply:
IMO Things should be enabled by default, if they're necessary for preventing developer confusion, and selectively disabled